astro/src/content/blog/announcing-fusionauth-1-50.mdx

```diff
@@ -0,0 +1,122 @@
+
+While there's a lot that goes into building third party ecosystems, one non-negotiable is clear permissions about what the applications can do. You also want to communicate to your end users the data or functionality an application is seeking before they grant it access.
+
+For your middle out compression startup, you create a set of scopes for your application:
+
+* `video:download`
+* `video:upload`
+* `video:delete`
+* `video:search`
+
+After that, document how developers can ask for these scopes using the `scope` parameter on their initial request. End users who want to install or use these applications will the be presented with a screen similar to this:
+
+![Consent screen](/img/blogs/release-1-50/consent-example.png)
+
+This screen is [themable and localizable](/docs/customize/look-and-feel/localization).
+
+If there's a third party application, working name `LOTRUS`, which looks up a video and modifies it to turn everyone into their favorite Lord of the Rings character, this application will need `video:download`, `video:search`, and `video:upload` permissions. On the other hand, a third party application which looks at videos in aggregate and performs statistics on them only needs `video:search.
+
+Assuming a user consents, FusionAuth would issue each of these applications a token with the appropriate scopes. When the application makes requests for data from the Pied Piper APIs, the APIs can check for the appropriate permissions before taking action.
+
+The ability to create custom scopes and consent screens allows you to safely and securely extend user data access to partners and third party developers.
+
+This release also brings FusionAuth into better compliance with a number of specifications such as OIDC. but don't worry, we've also built in backwards compatibility. The [Scopes documentation](/docs/lifecycle/authenticate-users/oauth/scopes) discusses how you might want to update your configuration.
+
+Prompting the user for consent is limited to certain paid plans. Learn more by reviewing [the pricing page](/pricing) or [contacting our sales team](/contact).
+
+## UserInfo Lambda
+
+The [UserInfo endpoint](/docs/lifecycle/authenticate-users/oauth/endpoints#userinfo), as defined in the OIDC specification, takes an access token and returns information about the user. Per the specification:
+
+> The UserInfo Endpoint is an OAuth 2.0 Protected Resource that returns Claims about the authenticated End-User.
+
+FusionAuth has supported this endpoint since inception, but this release now allows you to customize the response. You use a lambda, which is a snippet of JavaScript assigned to an application.
```

Comment:
```suggestion
FusionAuth has supported this endpoint since inception, but this release now allows you to customize the response. You use a [lambda](/docs/extend/code/lambdas), which is a snippet of JavaScript assigned to an application.
```
Might consider linking to the lamda over view page here.